design/gdd.md

```diff
@@ -200,21 +200,21 @@ Play [filling_form](sfx/filling_form.md)
 
 [marjorie](characters/marjorie.md), [calvin](characters/calvin.md) and [rebeca](characters/rebeca.md) stand in the [[locations/hall]], in front of the [front_door](items/front_door.md). We see a bagpack, a bag and a [suitcase](items/suitcase.md) in front of them. The bagpack belongs to [calvin](characters/calvin.md), the bag to [rebeca](characters/rebeca.md) and the [suitcase](items/suitcase.md) to [marjorie](characters/marjorie.md).
 
-They stand there for a few seconds, looking around. [marjorie](characters/marjorie.md) is frightened, she is taking in the uncommon surroundings. [rebeca](characters/rebeca.md) is annoyed, she is filling out a form.
+They stand there for a few seconds, looking around. [marjorie](characters/marjorie.md) is frightened, she is taking in the uncommon surroundings. [rebeca](characters/rebeca.md) is annoyed, and is filling out a form.
 
 [calvin](characters/calvin.md) calmly looks around.
 
 (beat) Stop sound effect.
 
 > [rebeca](characters/rebeca.md) (harsh)
 >
-> Living room is right this way, kitchen to the left, the bathroom and the bedrooms are upstairs. Your room is just across the stairs. Can't miss it.
+> Living room is right this way, kitchen to the left, the bathroom and the bedrooms are upstairs. Your room is just across the stairs. You can't miss it.
```

Comment:
I wanted to drop the "You" to make Rebeca act more distanced and annoyed.

Reply:
It's just that you normally don't cut the pronoun when speaking directly to a second person.

Reply:
Okay, agreed.